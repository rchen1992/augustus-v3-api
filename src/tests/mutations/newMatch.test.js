const { testQuery } = require('@tests/utils');

testQuery({
    id: 'Create new match',
    query: `
        mutation newMatch($input: NewMatchInput!) {
            newMatch(input: $input) {
                id
                tied
            }
        }
    `,
    variables: {
        input: {
            ladderId: 1,
            user1Id: 1,
            user2Id: 2,
            winnerId: 1,
            loserId: 1,
        },
    },
    context: () => {
        return {
            services: {
                match: {
                    newMatch() {
                        return {
                            match_id: 10,
                            ladder_id: 1,
                            user_1_id: 1,
                            user_2_id: 2,
                            winner_id: 1,
                            loser_id: 1,
                            tied: false,
                        };
                    },
                },
            },
        };
    },
    expected: undefined,
});
