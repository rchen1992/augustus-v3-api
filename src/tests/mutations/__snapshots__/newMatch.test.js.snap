// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Create new match 1`] = `
Object {
  "data": Object {
    "newMatch": Object {
      "id": "10",
      "tied": false,
    },
  },
}
`;
